frontend/src/network/notes_api.ts

```diff
@@ -41,3 +48,35 @@ export async function updateNote(
 export async function deleteNote(noteId: string) {
   await fetchData("/api/notes/" + noteId, { method: "DELETE" });
```

Comment:
nit: the name `fetchData` doesn't make sense here :/ we need something like `makeRequest` or `sendRequest`